### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by EmmanuelPetrus**
I've implemented the needed changes on the homepage. However, some of the widgets are overlapping which I'll have to consult with my front-end partner

### CODE REVIEW

File: pricejar/templates/navbar.html [hunk 1]
```diff
@@ -1,41 +1,224 @@
...
+    src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
+    crossorigin="anonymous"
+  ></script>
+</nav>
+
+{% else %}
+<nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#" style="padding: 10px">PriceJar</a>
```
Comment (mirera) [inline pricejar/templates/navbar.html]: Consider using the coloured pricejar text logo for the second part of unauntheticated user.

File: pricejar/templates/navbar.html [hunk 2]
```diff
@@ -1,41 +1,224 @@
...
+            <li><hr class="dropdown-divider" /></li>
+            <li>
+              <a class="dropdown-item" href="#">Something else here</a>
+            </li>
+          </ul>
+        </li>
+      </ul>
+      <ul class="navbar-nav justify-content-end">
+        <li class="nav-item">
+          <a href="login"><button class="btn" type="submit">Login</button></a>
```
Comment (mirera) [inline pricejar/templates/navbar.html]: update this urls for login and register.

**Review by mirera — CHANGES_REQUESTED** [review]

**Comment by EmmanuelPetrus** [comment]
executed the new functionalities requested

**Review by EmmanuelPetrus — COMMENTED** [review]
implemented the requested changes

**Pull request merged**